scripts/openyonclickinstall.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
-# To get OpenY on DigitalOcean 16.04 LST x64 droplet run the command:
-# bash < <(curl -Ls http://bit.ly/initopeny)
+# To get latest dev OpenY on DigitalOcean 16.04 LST x64 droplet run the command:
+#   curl -Ls https://raw.githubusercontent.com/AndreyMaximov/openy-project/8.1.x/scripts/openyonclickinstall.sh | bash -s
```

Comment:
by default we have to provide latest stable. not dev

Reply:
@podarok Sounds good. The wrong comment here. I'm fixing it.

Reply:
`stable` - the script installs the latest stable version of Open Y (e.g. 8.1.13.2), the default value

You can see this from the PR description

Reply:
@podarok fixed